src/commands/info/releasenotes/display.ts

```diff
@@ -22,8 +22,8 @@ import { parseReleaseNotes } from '../../../shared/parseReleaseNotes';
 // Initialize Messages with the current plugin directory
 Messages.importMessagesDirectory(__dirname);
 
-const HIDE_NOTES = 'PLUGIN_INFO_HIDE_RELEASE_NOTES';
-const HIDE_FOOTER = 'PLUGIN_INFO_HIDE_FOOTER';
+const HIDE_NOTES = 'SFDX_HIDE_RELEASE_NOTES';
```

Comment:
Just musing here, but how about renaming the variables to be more explicit about when customers can use them?   For example:

SFDX_HIDE_REL_NOTES_DURING_UPDATE
SFDX_HIDE_REL_NOTES_FOOTER_DURING_UPDATE
SFDX_REL_NOTES_UPDATE_DISPLAY_TIMEOUT

or

SFDX_UPDATE_HIDE_REL_NOTES
SFDX_UPDATE_HIDE_REL_NOTES_FOOTER
SFDX_UPDATE_REL_NOTES_TIMEOUT

Reply:
Also, should I doc that timeout env var?

Reply:
Your comments made me realize I should probably change some logic in this command. Thanks @jshackell-sfdc! Might not get to this until Monday 🦃

Reply:
Updated the env var names @jshackell-sfdc. Your call on documenting the TIMEOUT. I think it will be unlikely that someone will need to set it, but wanted it to be available if needed.